src/util/findAndMoveHelpers.js

```diff
@@ -89,7 +89,7 @@ const findHandleDataNearImagePoint = function(
       getProximityThreshold(interactionType, toolName)
     );
 
-    // custom property which will not allow for moving this annotation
+    // Custom property which will not allow for moving this annotation
```

Comment:
This comment should be changed to say `will not allow for resizing this annotation`, since it prevents adjusting from the handles. Otherwise it seems redundant with the `Prevents annotation from being moved` block.